data/infoy_bio.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Infoy",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "world(!successfull=try)",
+  "avatar": "https://avatars.githubusercontent.com/u/90406016?s=400&u=65d5649db9b66f4c2478e322842d5d01aad3172a&v=4",
```

Replace avatar image as follows
```suggestion
  "avatar": "https://github.com/PSGitHubUser1.png",
```